app/presenters/setting_presenter.rb

```diff
@@ -51,7 +71,7 @@ def readonly?
   end
 
   def value
-    SETTINGS.fetch(name.to_sym) { super }
+    SETTINGS.fetch(name.to_sym) { explicit_value? ? super : default }
```

Comment:
This should deal with `default = <some_default>` but user set nil (it will be fetched from database and set as explicit `value =` and the `@explicit_value = true` will be set.